.github/workflows/scripts/win_test.ps1

```diff
@@ -13,6 +15,10 @@ if ("$env:TI_WANTED_ARCHS".Contains("cuda")) {
     pip install "torch; python_version < '3.10'"
     pip install "paddlepaddle==2.3.0; python_version < '3.10'"
 }
+
+# Run C++ tests
+python tests/run_tests.py --cpp
```

Comment:
without `if (-not $?) { exit 1 }` the ps script seems not terminate upon errors on our CI.
```suggestion
python tests/run_tests.py --cpp
if (-not $?) { exit 1 }
```

Reply:
fixed